src/main/java/org/ecocean/api/UserHome.java

```diff
@@ -41,9 +43,35 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
             return;
         }
         home.put("user", currentUser.infoJSONObject(context, true));
-
-        // TODO ES replace
-
```

Comment:
this work should be moved to a **new api endpoint** per the ticket: `/api/v3/projects`. so `UserHome.java` should not be affected.

instead, create a new `ecocean/api/Projects.java` which will contain this code and handle the new endpoints.

for help on how to handle the ending of the api (`projects` vs `projects/all`) you can see [how UserInfo.java handles](https://github.com/WildMeOrg/Wildbook/blob/master/src/main/java/org/ecocean/api/UserInfo.java#L58-L62) the optional passed user guid: